source/reference/hyperlinks.txt

```diff
@@ -0,0 +1,102 @@
+.. _style-guide-hyperlinks:
+
+==========
+Hyperlinks
+==========
+
+You can link out to other sites, internal pages or specific sections on
+a page using inline hyperlinks and cross-referenced hyperlinks.
+
+Inline Hyperlinks
+-----------------
+
+Use inline hyperlinks when you want to link to a website with its full URL.
+
+Example
+~~~~~~~
+
+.. code-block:: rst
+
+   Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Rendered View
+`````````````
+
+Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Cross-Referenced Hyperlinks
+---------------------------
+
+Use cross-referenced hyperlinks when you want to link to another page in
+docs. This page can be in the same repo, or another within the ``MongoDB``
+organization.
+
+Procedure
+~~~~~~~~~
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Check if the cross-referenced hyperlink exists.
+
+      If the cross-referenced hyperlink is in a different MongoDB repo, 
+      add the documentation URL landing page and append
+      ``/objects.inv``. Then, add the URL to the  ``intersphinx`` array
+      in the ``snooty.toml`` file and skip the next step.
+
+      For example, if you want to use hyperlinks in the manual, you'd
+      add this to the ``intersphinx`` array:
+
+      .. code-block:: rst
+
+         https://www.mongodb.com/docs/manual/objects.inv
+
+      If someone else already created the cross-referenced hyperlink in
+      the same repo, skip the next step.
+   
+   .. step:: Create the cross-referenced hyperlink.
+      
+      Place your cross-referenced hyperlink a line before where you 
+      want the reference to be.
+
+      .. note::
+
+         You need to place this two lines before where you want the link
+         to.
```

Comment:
I'm still not understanding what to do here. Is the "cross-referenced hyperlink" the objects.inv URL? Or where you actually put the link in text? It's also a little confusing that the step says "a line before" while the note says "two lines before." I'm also not sure which file we're working in--snooty.toml or the text file? Maybe an example would make things clearer?